docs/contribute/benchmarks/TRAINING/benchmarks_DL2_EnergyLUT.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Not a comment on this notebook, but in general: these areas where energy reconstruction doesn't work well would make a good cut on events later (e.g. remove events in these regions, or mark them as another event class for their own IRF)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/protopipe/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I'll add this comment as a markdown cell so we don't forget (and hopefully there are other people looking at these notebooks aside from us...)</p>

Reply:
<p>DONE</p>